composer/callbacks/run_directory_uploader.py

```diff
@@ -202,6 +211,11 @@ def _trigger_upload(self, logger: Optional[Logger], log_level: Optional[LogLevel
             logger.metric(log_level, {"run_directory/uploaded_files": files_to_be_uploaded})
 
 
+def get_obj_name_for_local_file(object_name_prefix: str, local_filepath: str) -> str:
```

Comment:
Curious why this change?

Reply:
I wanted to lift the logic for creating the object name by ripping out the run directory ad attaching the prefix out so that it could be imported by the trainer as well